canonical/react-components
# Add Segmented control component

## Done

- Porting Segmented Controls into React Components

## QA

### Storybook

To see rendered examples of all react-components, run:

```shell
yarn start
```

### QA in your project

from `react-components` run:

```shell
yarn build
npm pack
```

Install the resulting tarball in your project with:

```shell
yarn add <path-to-tarball>
```

### QA steps

- Visit https://react-components-896.demos.haus/?path=/story/segmented-control--default-story
- Verify everything works

## Fixes

Fixes: # .
